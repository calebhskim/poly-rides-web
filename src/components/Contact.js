import React from 'react';
import { View, Text } from 'react-native';

import Paper from 'material-ui/Paper';

import cardStyle from '../styles/components/card';

const Contact = () => (
  <Paper style={cardStyle} id='contact'>
    <View>
      <Text>Contact</Text>
    </View>
  </Paper>
);

export default Contact;
