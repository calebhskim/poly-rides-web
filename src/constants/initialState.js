import lifecycles from './lifecycles';

export default {
  auth: {
    error: {},
    lifecycle: lifecycles.AUTH_NOT_LOGGEDIN,
    user: {
      displayName: null,
      email: null,
      emailVerified: false,
      isAnonymous: null,
      photoURL: null,
      refreshToken: null,
      uid: null,
    },
    fbToken: null,
  },
  config: {
    title: 'PolyRides',
  },
  firebase: {
    app: null,
    fbProvider: null,
  },
};
