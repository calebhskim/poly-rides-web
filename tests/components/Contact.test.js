import React from 'react';

import configureMockStore from 'redux-mock-store';
import { mount, shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import Contact from '../../src/components/Contact';
import initialState from '../../src/constants/initialState';

const mockStore = configureMockStore();

describe('<Contact />', () => {
  it('renders properly', () => {
    const store = mockStore(initialState);
    const wrapper = shallow(<Contact store={store} />).shallow();

    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
