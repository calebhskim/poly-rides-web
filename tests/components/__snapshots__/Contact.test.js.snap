// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Contact /> renders properly 1`] = `
<Paper
  circle={false}
  id="contact"
  rounded={true}
  style={
    Object {
      "display": "flex",
      "height": "100vh",
    }
  }
  transitionEnabled={true}
  zDepth={1}
>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
    >
      Contact
    </Text>
  </View>
</Paper>
`;
